src/server/pegasus_server_impl.cpp

```diff
@@ -274,6 +281,12 @@ void pegasus_server_impl::on_get(get_rpc rpc)
     resp.partition_index = _gpid.get_partition_index();
     resp.server = _primary_address;
 
+    if (!_read_size_throttling_controller->get_token(FLAGS_read_throttling_by_size_request_count)) {
+        rpc.error() = dsn::ERR_BUSY;
+        _counter_recent_read_throttling_reject_count->increment();
```

Comment:
I suggest the reject count can be distinct the `size` and `qps`, which we can get the request why exceed the throld

Reply:
ref https://github.com/XiaoMi/rdsn/blob/master/src/replica/replica_throttle.cpp#L56
For compatibility,  I think it's ok to combine them. We can distinguish `size` and `qps` from other perf_counters easily

Reply:
Next pr consider it, now is ok